Tasks/AndroidSigning/Strings/resources.resjson/en-US/resources.resjson

```diff
@@ -3,20 +3,20 @@
   "loc.helpMarkDown": "[More Information](https://go.microsoft.com/fwlink/?LinkID=613717)",
   "loc.description": "Sign and align Android APK files",
   "loc.instanceNameFormat": "Signing and aligning APK file(s) $(files)",
-  "loc.group.displayName.jarsignerOptions": "Jarsigner Options",
+  "loc.group.displayName.jarsignerOptions": "Signing Options",
   "loc.group.displayName.zipalignOptions": "Zipalign Options",
   "loc.input.label.files": "APK Files",
   "loc.input.help.files": "Relative path from the repo root to the APK(s) you want to sign. You can use wildcards to specify multiple files. For example, `**/bin/*.apk` for all .APK files in the 'bin' subfolder.",
   "loc.input.label.jarsign": "Sign the APK",
-  "loc.input.help.jarsign": "Select if you want to sign the APK with a provided keystore file.",
+  "loc.input.help.jarsign": "Select this option to sign the APK with a provided keystore file. Unsigned APKs can only run in an emulator. APKs must be signed to run on a device.",
   "loc.input.label.keystoreFile": "Keystore File",
-  "loc.input.help.keystoreFile": "The keystore file should be entered as a file path. It can either be checked into source control or placed on the build machine directly by an administrator. It is recommended to encrypt the keystore file in source control and use the 'Decrypt File' task to decrypt the file during the build.",
+  "loc.input.help.keystoreFile": "Enter the file path to the keystore file that should be used to sign the APK. It can either be checked into source control or placed on the build machine directly by an administrator. It is recommended to encrypt the keystore file in source control and use the 'Decrypt File' task to decrypt the file during the build.",
```

Comment:
Can we change the keystoreFile inputType to filePath if it is not already? It might need some code change and validation but should be straightforward